### ISSUE TITLE
Add support for graphql nested syntax

### ISSUE THREAD

**Issue by Janpot** [bathos/Ecmascript-Sublime#58]
As title says 🙂 
I'd say `graphql` tagged templates are quite common in javascript code

**Comment by blake-regalia** [comment]
Sure! The problem though is that as of now there only seems to be one GraphQL syntax and the scopes that it creates look terrible in our color schemes, we actually have this problem with a few other syntaxes but this one in particular is really ugly:
<img width="388" alt="screen shot 2018-09-19 at 8 11 38 am" src="https://user-images.githubusercontent.com/1456400/45762705-d3316480-bbe3-11e8-85ec-2f60bb191b98.png">

I would suggest we'd first need to make a bit of effort to add each scope to (at least) one of the color schemes (likely excelsior since this is the one that supports nested syntaxes right now).

**Comment by Janpot** [comment]
@blake-regalia I also just realised I often use `lua` in javascript for scripting with redis. Do you think that would be valuable target as well for nested syntax?

**Comment by blake-regalia** [comment]
@Janpot absolutely. Lua syntax comes with ST3 and their scopes tend to look more natural in the color scheme so that should be no problem.

**Comment by blake-regalia** [comment]
@Janpot [v1.8.0](https://github.com/bathos/Ecmascript-Sublime/releases/tag/1.8.0)

**Comment by blake-regalia** [comment]
Should also add highlighting support for template tag `gql` from  https://github.com/apollographql/graphql-tag
